setup.py

```diff
@@ -18,12 +18,12 @@
     'compress-pickle>=1.2.0,<2',
     'humanfriendly>=8.2,<9',
     'numpy>=1.18.0,<2',
-    'pandas>=1.1,<1.1.5',
+    'pandas>=1.1,<2',
```

Comment:
Huh, I'm surprised this works, I thought SDV had some issue with pandas 2.0, which is why we didn't upgrade it sooner.

Did you check that the other libraries work with pandas 2?

Reply:
I updated Copulas to use the latest version of pandas (1.3.4) here https://github.com/sdv-dev/Copulas/pull/279 and all the tests passed. I saw that you had mentioned some test case failures in Copulas but I couldn't reproduce it for some reason.

Reply:
I suspect that the problem in Copulas happened only in one specific version, which I think was `1.1.5`, because with later versions the problem seemed to disappear. But the strange thing is that now 1.1.5 does not seem to fail either when forced, so we can just forget about it and move on for now as @katxiao is suggesting.